am-lib/src/main/java/uk/gov/hmcts/reform/amlib/AccessManagementService.java

```diff
@@ -127,7 +136,7 @@ public void revokeResourceAccess(ExplicitAccessMetadata explicitAccessMetadata)
      * @param userRoles accessor roles
      * @param resources envelope {@link Resource} and corresponding metadata
      * @return envelope list of {@link FilterResourceResponse} with resource ID, filtered JSON and map of permissions
-     *     if access to resource is configured, otherwise null.
+     *      if access to resource is configured, otherwise null.
```

Comment:
Why is that change there? Please try to avoid unnecessary changes that does nothing. It just takes time from reviewer.

Reply:
checkstyle wanted it moved.

Reply:
Checkstyle expects 4 whitespaces while you got 5. Please revert as previous state should satisfy style check.

Reply:
👍